lib/qa/authorities/linked_data/config.rb

```diff
@@ -32,7 +32,7 @@ def search
       end
 
       def term
-        @term ||= Qa::Authorities::LinkedData::TermConfig.new(authority_config.fetch(:term), prefixes)
+        @term ||= Qa::Authorities::LinkedData::TermConfig.new(authority_config.fetch(:term), prefixes, self)
```

Comment:
Passes in the full config so term config can delegate methods like `authority_name`.